src/theme/components/index.ts

```diff
@@ -1,26 +1,65 @@
 import type { Components } from '@mui/material/styles';
```

Comment:
avoid creating too many folders. The `components` folder can be dissolved and this file can live in the theme folder by nt name `components.tsx`. We know that its inside the theme folder and that is more than enough context

Reply:
we can edit. No need to follow that